app/subscriber/src/features/my-reports/edit/settings/template/charts/ChartPicker.tsx

```diff
@@ -99,15 +105,15 @@ export const ChartPicker: React.FC<IChartSectionProps> = ({ sectionIndex, chartI
 
         <Col className="frm-in">
           <label>Axis</label>
-          <FormikCheckbox
+          <Checkbox
```

Comment:
Fixed issue with `isHorizontal` because it's the opposite of the checkbox label.